src/hooks/useBeamer.ts

```diff
@@ -0,0 +1,33 @@
+import { useEffect } from 'react';
+
+declare const Beamer: any;
+
+export const useBeamer = (): void => {
+  useEffect(() => {
+    const initializeBeamer = () => {
+      if (typeof Beamer !== 'undefined') {
+        Beamer.init();
+      }
+    };
+
+    initializeBeamer();
+  }, []);
+};
+
+export const useWhatsNew = (
```

Comment:
as mentioned in a different comment, maybe we can move this and `useFeaturesBugs` into a utils file since they are not hooks and we can call them anywhere